lms/static/js/student_account/enrollment.js

```diff
@@ -35,6 +35,11 @@
                         // If so, redirect to a page explaining to the user
                         // why they were blocked.
                         this.redirect(responseData.user_message_url);
+                    } else if (jqXHR.status === 409) {
```

Comment:
why not declare a constant for `409` that can replace the comment or shorten it?
i.e `const USER_ALREADY_ENROLLED = 409`

Reply:
in above statements there was no constant used but rather a comment was done so did it for uniformity